api/connectionmanager/model.go

```diff
@@ -214,3 +214,36 @@ type ConnectionCount struct {
 type IDstruct struct {
 	ID string `json:"id"`
 }
+
+type UebaInternalModelInstance struct {
+	ID                string `json:"id" validate:"uuid,omitempty"`
```

Comment:
imho we do not need the `omitempty` tag since we're consuming the JSON (i.e structs are used in GET method) unless I am mistaken